<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LIN-BRCPTN3" failures="1" tests="8" name="SauceDemoTests" time="59.394" errors="0" timestamp="2025-10-23T16:09:01 IST">
  <testcase classname="com.saucedemo.tests.LoginTest" name="invalidLoginTest" time="0.586"/>
  <testcase classname="com.saucedemo.tests.LoginTest" name="validLoginTest" time="2.212"/>
  <testcase classname="com.saucedemo.tests.InventoryTest" name="verifyProductCount" time="0.493"/>
  <testcase classname="com.saucedemo.tests.ProductDetailsTest" name="verifyProductDetails" time="0.955"/>
  <testcase classname="com.saucedemo.tests.CartTest" name="verifyCartPageLoads" time="1.115"/>
  <testcase classname="com.saucedemo.tests.CheckoutInfoTest" name="fillCheckoutInformation" time="0.867"/>
  <testcase classname="com.saucedemo.tests.CheckoutOverviewTest" name="verifyTotalPrice" time="0.608"/>
  <testcase classname="com.saucedemo.tests.CheckoutCompleteTest" name="verifyOrderConfirmation" time="0.581">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.saucedemo.tests.CheckoutCompleteTest.verifyOrderConfirmation(CheckoutCompleteTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyOrderConfirmation -->
</testsuite> <!-- SauceDemoTests -->
